docs-ref-conceptual/install-azure-cli-apt.md

```diff
@@ -38,6 +38,7 @@ for the Azure CLI. This package has been tested with:
 3. Install the CLI:
 
    ```bash
+   sudo apt-get update
    sudo apt-get install apt-transport-https
    sudo apt-get update && sudo apt-get install azure-cli
```

Comment:
Since `apt-get update` is now earlier in the script, it can be removed here.

Reply:
Thanks, also I noticed `apt-get install` can now be collapsed

Reply:
Do you think after installing `apt-transport-https` doing an update might still be necessary? I guess I'm not sure...